core/prisma/schema/schema.prisma

```diff
@@ -442,6 +443,13 @@ model ActionRun {
   updatedAt        DateTime          @default(now()) @updatedAt
   PubValueHistory  PubValueHistory[]
 
+  // action run that triggered this action run
+  triggeringActionRunId String?
+  triggeringActionRun   ActionRun? @relation("triggering_action_run", fields: [triggeringActionRunId], references: [id], onDelete: SetNull)
```

Comment:
i want to display that an action was triggered by another, so i added this column. 

this is different from the initial idea of adding some kind of `stackId`, which would allow us to get all the runs in one stack.

the problem with this is that we would not be able to 
a. show the order of executions (except by time i guess)
b. see the separate threads if one action triggers multiple other actions, eg  `1->2, 1->3, 2->4, 3->5`. these would all have the same `stackId`